cfp/models.py

```diff
@@ -12,6 +17,20 @@ class Proposal(models.Model):
          (6, 'Performant Python'),
          (7, 'Scalable Python'),
          (8, '/etc')])
-
+    speaker_bio = models.TextField(verbose_name="Speaker Bio", max_length=500)
```

Comment:
It was 500 in the last year's cfp form. I copied it from the pybay2016 website but feel free to change it.